CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ You can find its changes [documented below](#060---2020-06-01).
 ### Highlights
 
 ### Added
+- Implementation of `Data` trait for `i128` and `u128` primitive data types. ([#1061] by @koutoftimer)
```

Comment:
This should go at the bottom of the added list and you need to add the PR link and your username link to their respective lists for the links to function